src/3rdparty/walkontable/src/overlay/bottom.js

```diff
@@ -165,12 +165,20 @@ class BottomOverlay extends Overlay {
     const scrollbarWidth = masterHolder.clientWidth === masterHolder.offsetWidth ? 0 : getScrollbarWidth(this.wot.rootDocument);
```

Comment:
This can be replaced to simpler assigning?
```suggestion
     const scrollbarWidth = getScrollbarWidth(this.wot.rootDocument);
```